MapboxDirectionsTests/V5Tests.swift

```diff
@@ -60,7 +60,7 @@ class V5Tests: XCTestCase {
         }
         XCTAssertNotNil(task)
         
-        waitForExpectations(timeout: 2) { (error) in
+        waitForExpectations(timeout: 5) { (error) in
```

Comment:
Even locally, 2 seconds would fail 1/16 times for the 3.2 MB GeoJSON

Reply:
Yikes, somehow I didn’t realize our test fixtures were that big. This isn’t a performance test, so we should probably think about breaking this test case up into minimal test cases the next time we need to fix it.